form-samples/two-column-logo-header/two-column-logo-header.json

```diff
@@ -0,0 +1,114 @@
+              "width": "100%",
+              "text-align": "left",
+              "padding": "21px 12px",
+              "overflow": "hidden"
+          },
+          "children": [
+              {
+                  "elmType": "div",
+                  "style": {
+                      "display": "flex",
+                      "flex-direction": "row"
+                  },
+                  "children": [
+                      {
+                          "elmType": "div",
+                          "style": {
+                              "width": "100%",
+                              "display": "flex",
+                              "flex-direction": "column",
+                              "align-items": "left",
+                              "margin": "0px"
+                          },
+                          "children": [
+                              {
+                                  "attributes": {
+                                      "class": "ms-fontSize-24",
+                                      "iconName": ""
+                                  },
+                                  "elmType": "div",
+                                  "txtContent": "='A New Improvement Request for ' + [$Title]",
+                                  "style": {
+                                      "margin-top": "5px"
+                                  }
+                              },
+                              {
+                                  "elmType": "div",
+                                  "children": [
+                                      {
+                                          "elmType": "a",
+                                          "class": "sp-field-quickActions",
+                                          "attributes": {
+                                              "class": "ms-fontSize-16 ms-fontColor-white",
+                                              "iconName": "=if([$RequestStatus] == 'Approved', 'SkypeCircleCheck', if([$RequestStatus] == 'Pending BPO Review', 'ProgressRingDots', if([$RequestStatus] == 'Rejected', 'StatusErrorFull', if([$RequestStatus] == 'Pending BE Review', 'ProgressRingDots','ProgressRingDots'))))"
+                                          }
+                                      },
+                                      {
+                                          "elmType": "span",
+                                          "attributes": {
+                                              "class": "ms-fontSize-16",
+                                              "iconName": ""
+                                          },
+                                          "style": {
+                                              "padding-right": "8px",
+                                              "padding-left": "8px"
+                                          },
+                                          "txtContent": "Request Status: Some Status"
```

Comment:
If you want to display the value of the `RequestStatus` column, you can do so by writing.

```
"txtContent": "='Request Status: '+[$RequestStatus]"
```

![image](https://github.com/pnp/List-Formatting/assets/51254139/188de09d-76c1-44bf-a2f4-76221ca96e17)

(If you do not want the values in the `RequestStatus` column to be displayed, no modification is necessary.)

Reply:
Added